### DISCUSSION TITLE
Microsoft Azure and Oqtane 5.1

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4133] [General]
I have many Oqtane deployments on Microsoft Azure... so I just want to share some of my experiences from this past week...

1. When using Static Server Rendering with Oqtane 5.1 you need to run it on the latest .NET SDK / Hosting Bundle (ie. 8.0.3 or 8.0.4). If you run it on an older SDK / Hosting Bundle you will sometimes encounter strange run-time behavior with Blazor.

3. By default an Azure App Service is currently running the 8.0.1 SDK (which has some known Blazor issues when used with Oqtane). You can determine the version of the SDK you are running by using Kudu in the Development Tools / Advanced Tools:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/38c382fd-5e91-4cf3-9106-649eec264f04)

Choose CMD from the Debug console menu and enter dotnet --info

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/83b23e01-955c-4e76-9bb1-6c1fdadc2949)

5. You can upgrade your App Service by using the Development Tools / Extensions, clicking Add, and searching for "ASP.NET Core" (note that you need to have at least a B1 subscription to install Extensions):

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/3835d801-5d91-4cf6-adf9-a2f876c49eb1)

6. You will want to choose the correct version of the SDK based on your Platform setting. By default the Platform setting for Azure App Services is 32 Bit. You can change it to 64 Bit in the Configuration / General settings tab:
 
![image](https://github.com/oqtane/oqtane.framework/assets/4840590/c435ab7c-dc78-469e-8d3b-83a3d0851470)

**Comment by ristopaarni** [comment]
Please make similar case study for other extensions. The update function on azure destroys the site? Is that still the case?

**Reply by sbwalker** [reply]
@ristopaarni I do not understand what you are requesting? What do you mean by "the update function on Azure" - there are many UI options in the Azure portal which allow you to save information related to your environment.
